en/micro-integrator/docs/references/connectors/as400-pcml-connector/as400-pcml-connector-configuration.md

```diff
@@ -0,0 +1,65 @@
+# Setting up the AS400 PCML Environment  
+
+The AS400 PCML connector allows you to access RPG programs that are available on AS400 (renamed as IBM iSeries) servers using WSO2 EI. This is done using [Program Call Markup Language](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/rzahh/pcml.htm) (PCML). 
+
+To work with the AS400 PCML connector, you need to have a IBM iSeries server. If you do not have a IBM iSeries server, go to [public IBM iSeries server](https://pub400.com/), and create a public IBM iSeries server.
+
+The connector uses the IBM JTOpen library for all its operations. Copy the **jt400.jar** to `<ESB_HOME>/repository/components/lib` folder. You can download the IBM JTOpen library from (here)[https://sourceforge.net/projects/jt400/].
+
+### Setting up public IBM iSeries server
+
+Follow the steps below to create public IBM iSeries server.
+
+1. Go to [https://pub400.com](https://pub400.com/) and click **Sign up** button.
+
+    <img src="../../../../assets/img/connectors/as400-signup.png" title="AS400-signup page" width="600" alt="AS400-signup page"/>
+    
+2. Enter required details for request a new user profile and click **Submit** button.
+
+    <img src="../../../../assets/img/connectors/as400-signup-details.png" title="AS400-signup details" width="600" alt="AS400-signup details"/>
+    
+3. You will received the reply email with the client credentials to the your email account (It will take some hours).
+
+4. Navigate in to the [IBM i Access - Client Solutions](https://www.ibm.com/support/pages/ibm-i-access-client-solutions) page.
+
+5. Download the **IBM i Access Client Solutions**.
```

Comment:
Shall we first mention why we need IBM i Access Client Solutions?

Reply:
Added description.